terraform/modules/auth_service/secrets.tf

```
@@ -7,6 +7,16 @@ data "aws_secretsmanager_secret" "saml_certificate" {
   name = "api-saml-certificate-${var.environment}"
 }
 
+# TODO 836 ticket specific release action: manually create these secrets, copying the current values from the API secrets, so they can be referenced here
+# See api/README.md and terraform/test/main.tf in the Delta repository for setup instructions
+data "aws_secretsmanager_secret" "delta_saml_private_key" {
+  name = "delta-saml-private-key-${var.environment}"
+}
+
+data "aws_secretsmanager_secret" "delta_saml_private_key" {
```

```suggestion
data "aws_secretsmanager_secret" "delta_saml_certificate" {
```
should create these at least in test before merging

---

I created the Test ones, not the others yet.